workflows/Get_InsightVM_Asset_Details_And_Scan_Asset_From_InsightConnect/workflow.spec.yaml

```
@@ -0,0 +1,25 @@
+extension: workflow
+products: ["insightconnect", "insightvm"]
+name: Get_InsightVM_Asset_Details_And_Scan_Asset_From_InsightConnect
```

Need to update `name` and directory name to match convention in the workflow repo
```suggestion
name: Get_InsightVM_Asset_Details_and_Scan_Asset_from_InsightConnect
```